contentcuration/contentcuration/management/commands/runserver.py

```diff
@@ -9,31 +9,31 @@
 
 class Command(RunserverCommand):
```

Comment:
I would suggest deleting this, and borrowing the `npm-run-all` approach we are now using in Kolibri.

Reply:
What's the motivation for this, outta curiosity?

Reply:
The command was created to run the JS build process when the devserver is invoked.

It sometimes fails to stop when the devserver stops though.

Reply:
Ah alright. Given that we only ever have 1 JS script (build), do you think `npm-run-all` is a better fit than a `Make` command separated by `;`?

Reply:
Pretty sure that would run them sequentially, rather than in parallel, which would mean the second would only run after the first had quit, no?

Reply:
Ah, I'd have to use a `&` instead. I can see this getting messy pretty quick. @aronasorman thoughts?

Reply:
How would `npm-run-all` be run when we run the dev server? Under another line in `make devserver`?

Reply:
This is how we use it in Kolibri: https://github.com/learningequality/kolibri/blob/develop/package.json#L16

If there is an equivalent way to launch two processes in parallel in make, probably worth doing that!

Reply:
Going with `npm-run-all`. Bash includes really simple ways to parallelize, but no straightforward way to kill all the parallel processes without messing with fore/background stuff. There are other commands that provide simple means of doing both, but I think it'd be better to have a new JS dependency than a system dependency.

Reply:
Kept the file there because it was doing some stuff with minio that I admittedly don't understand. Took out all the webpack stuff, though!